lib/facter/gluster.rb

```diff
@@ -78,11 +73,12 @@
           end
         end
       end
+      # rubocop:disable Metrics/BlockNesting
       if volume_options
         volume_options.each do |vol, opts|
           Facter.add("gluster_volume_#{vol}_options".to_sym) do
             setcode do
-              opts.join(',')
+              opts.join(';')
```

Comment:
Do you think this counts as a breaking change?  ie Outside its use in this module, users might be using this fact in their own profiles??  Now that we've dropped puppet 3 support, all users should have a version of facter that supports [structured-facts](https://docs.puppet.com/facter/3.6/fact_overview.html#writing-structured-facts)  This means the fact could be changed to just return the array - getting rid of the `join` and corresponding `split`?